src/lint/linter/ArcanistCourserorgLinter.php

```diff
@@ -42,18 +42,22 @@ public function getLintNameMap() {
   }
 
   public function lintPath($path) {
+    $absolutePath = realpath($path);
+    $webPath = $_SERVER['HOME'].'/base/coursera/web/';
+    $escapedWebPath = preg_quote($webPath, '/');
```

Comment:
could you instead make a `$relPath` and use it in the existing preg_match calls?